resources/views/help/developer.twig

```diff
@@ -0,0 +1,127 @@
+{% extends "layouts.master" %}
+
+{% block title %}Help for developers{% endblock %}
+
+{% block content %}
+<ol class="breadcrumb">
+    <li>{{ link_to_route('projects.index', 'Home') }}</li>
+    <li class="active">Help for developers</li>
```

Comment:
I'd split this up to "Help / Developers" where Help is a link to main Help page (route 'help').